lib/model/__init__.py

```diff
@@ -1,33 +1,72 @@
 import time
 from sklearn.metrics import f1_score
+from sklearn.grid_search import GridSearchCV
 
 class Model:
-    def __init__(self, classifier, parameters={}):
-        self.classifier = classifier(**parameters)
+    def __init__(self, classifier, parameters={}, gs_params={}):
+        self.classifier = classifier
+        self.model = classifier(**parameters)
```

Comment:
+1 Is true ... the classifier is the classifier and the model is the model. Nice!